Clinical-Genomics/cgweb
# show add samples btn if rml

How to test:
0. Install branch on stage on clinical-db (servers/resources/sh ./update-cgweb-stage.sh)
1. Login on clinical-stage
2. Click "New order"
3. Chose an order type (RML)
4. Save
Expected outcome:
A button to add family/sample should appear for every type, the Add family button is not usable until customer is selected and saved
